lib/resources/pages/register_page.dart

```diff
@@ -21,6 +22,14 @@ class _RegisterPageState extends NyState<RegisterPage> {
     super.init();
   }
 
+  @override
+  void dispose() {
+    super.dispose();
+  }
+
+  bool isChecked1 = false;
+  bool isChecked2 = false;
```

Comment:
Bunlarin isimlerini daha anlamli bir sey yapabiliriz, tcCheckbox (terms and conditions) ve truthAckCheckbox (dogruluk beyani) gibi.